pkg/webhook/add_mutating.go

```diff
@@ -0,0 +1,10 @@
+package webhook
+
+import (
+	"github.com/nmstate/kubernetes-nmstate/pkg/webhook/mutating"
```

Comment:
I think the webhook should go under pkg/webhook/nodenetworkconfigurationpolicy, based on the type. It is more important than the fact we are changing it. But maybe it is just a matter of taste, up to you.